www/source/docs/container-orchestration-mesos.html.md

```diff
@@ -22,23 +22,29 @@ You can create native Mesos containers from Habitat packages by following these
 
 2. Install or [build](/docs/create-packages-build) the Habitat package from which you want to create a Marathon application, for example:
 
-       hab pkg install yourorigin/yourpackage
+    ```
```

Comment:
We should port this entire article into a blog with the date is was published. Doesn't read like docs. Reads like something written before we had a blog to put it in.